.gitlab-ci.yml

```diff
@@ -439,6 +439,32 @@ TF_Service::package:
     - unity
     - win10
 
+TF_Service::dotNet::ARMLinux::package:
+  stage: package
+  image: ubuntu:20.04
+  dependencies:
+    - TF_Service::dotNet::ARMLinux::build
+  needs: [ "TF_Service::dotNet::ARMLinux::build" ]
+  script:
+    - ln ./Service_dotNet_Build/libatomic.so.1.2.0 ./Service_dotNet_Build/libatomic.so.1
+    - ln ./Service_dotNet_Build/libLeapC.so.5 ./Service_dotNet_Build/libLeapC.so
+    - ln ./Service_dotNet_Build/libstdc++.so.6.0.28 ./Service_dotNet_Build/libstdc++.so.6
+    - chmod +x ./Service_dotNet_Build/TouchFree_Service
+
+    - cd "$CI_PROJECT_DIR/Scripts/BrightSign_Scripts/"
+    - chmod +x ./make-extension-ubi
+    - apt-get update
+    - apt-get --assume-yes install squashfs-tools
+    - ./make-extension-ubi "$CI_PROJECT_DIR/Service_dotNet_Build/" TF_Service_BS
+  artifacts:
+    name: "TouchFree_Service_Extension"
+    paths:
+      - "$CI_PROJECT_DIR/Scripts/BrightSign_Scripts/build-TF_Service_BS"
```

Comment:
The artefacts end up several directories deep in the zip, could we bring them up to the top level to make it a bit tidier?